doc/glossary.rst

```diff
@@ -0,0 +1,32 @@
+Glossary
+========
+
+.. currentmodule:: hnn_core
+
+The Glossary provides short definitions of HNN-specific vocabulary and
+general computational neuroscience concepts. If you think a term is missing,
+please consider creating a new issue or opening a pull request to add it.
+
+.. glossary::
+    :sorted:
+
+
+    drive
+        An artificial cell or population of artificial cells that only produces
+        spikes according to a given pattern and is used to initiate and sustain
+        cell spiking in the network
+
+    gid
+        The "global" cell ID. Global refers to the fact that the cell ID
+        is unique even if the simulation is split across cores.
+
+    distal
+        Anatomical location that is further away from the soma.
+        Distal drives refer to extrinsic connections to the distal dendrites of
+        the cortical column, which often originate from direct thalamic nuclei.
+
+    proximal
+        Anatomical location that is closer to the soma.
+        Proximal drives refer to extrinsic connections to the proximal dendrites
+        of the cortical column, which often originate from indirect thalamic
+        nuclei, as well as other cortical regions.
```

Comment:
```suggestion
        of the cortical column, which often originate from direct thalamic
        nuclei.
```

Reply:
is "direct"/"indirect" an anatomical location? I often get confused by this kind of phrasing that seems quite common on neuroscience ... you google "direct thalamic nuclei" and you get "direct pathway" and "thalamic nuclei" but no "direct thalamic nuclei" ...

Reply:
> is "direct"/"indirect" an anatomical location? I often get confused by this kind of phrasing that seems quite common on neuroscience ... you google "direct thalamic nuclei" and you get "direct pathway" and "thalamic nuclei" but no "direct thalamic nuclei" ...

I'm not convinced that "direct" and "indirect" are the proper terms to use here. Maybe we should stick with "lemniscal" and "non-lemniscal" as described in the HNN-GUI tutorials.

This distinction does not separate two anatomical regions, rather, it's an informal classification scheme that some anatomists use to categorize thalamic nuclei based on their afferent projections into cortex. For instance, the ventral posteromedial nucleus (VPM) is part of the lemniscal pathway that has bulk projections to the granular layer of S1. Conversely, the posterior medial nucleus (POm) has dispersed projections (that are not part of the lemniscal pathway) to numerous cortical regions that happen to include the superficial layers of S1. This might be a good place for @stephanie-r-jones to weigh in.